### DISCUSSION TITLE
Launching cartridges through SideKick20

### DISCUSSION THREAD

**Discussion by vicxlst** [frntc/Sidekick64#101] [Q&A]
Hi, sorry if this is duplicated elsewhere, i did look through but couldn't find the topic. I could use a little primer on launching cartridge images if possible.

I own a copy of StarBattle but have no means to dump it so I've gotten a few images of sorts off the internet.

So far as i can discern i have two files, both 8k with a .CRT extension. When i try to launch either one i get Error: NO .CRT - FILE

I appreciate i'm probably making some fundamental mistakes here but could use a little primer on what to do?

I see in the help it makes mention of selecting multi part but i could not find the character on the keyboard to select the two images in case that's what it was.

I can run .PRG and D.64 fine and really impressed with the HDMI output.

**Comment by vicxlst** [comment]
Hi, 

after a bit of digging i found the answer here:

https://www.forum64.de/index.php?thread/130626-sidekick20-aufbau-fragen/&pageNo=3

There's quite a lot of cartridges that initially launch but then ditch back to ready screen during gameplay or crash/hang. 

going to check different PRG files.

**Comment by RaymondDay61** [comment]
You need some invite to download from that link you gave.
